presto-main/src/main/java/com/facebook/presto/sql/planner/LocalExecutionPlanner.java

```diff
@@ -2050,6 +2051,14 @@ private PhysicalOperation createLookupJoin(JoinNode node,
                 PhysicalOperation probeSource,
                 LocalExecutionPlanContext context)
         {
+            // Determine if planning broadcast join
+            boolean isBroadcastJoin = false;
```

Comment:
you can make this one line

`isBroadcastJoin = distributionType.siPresent() && distributionType.get() == REPLICATED`

also, static import JoinNode.DistributionType.REPLICATED